apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/ManagedErrorActivity.java

```diff
@@ -72,7 +77,9 @@ protected void send(View view) {
                 Method method = Crashes.class.getDeclaredMethod("trackException", Throwable.class, Map.class);
                 method.setAccessible(true);
                 method.invoke(null, t, properties);
-
+                AttachmentsUtils.getInstance().setFileAttachment(sSharedPreferences.getString(FILE_ATTACHMENT_KEY, null));
```

Comment:
I don't understand the logic here. `AttachmentUtils` is a singleton and stores attachments in memory. 
But they are retrieved from preferences only here? And where are they saved to preferences or deleted from them?
If it's only [in MainActivity](https://github.com/microsoft/appcenter-sdk-android/blob/develop/apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java#L108), then I'd rather rewrite this. The logic of working with them is all over the place. We don't need a singleton, and `AttachmentUtils` class can work with `SharedPreferences`.